libs/design/src/molecules/feature/feature-body/feature-body.component.ts

```diff
@@ -0,0 +1,12 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: '[daff-feature-body]',
+  template: '<ng-content></ng-content>',
+  host: {'class': 'daff-feature__body'}
+})
```

Comment:
`changeDetection: ChangeDetectionStrategy.OnPush`

Reply:
Resolved in 688add3